index.base.html

```diff
@@ -0,0 +1,198 @@
+
+    <div class="container-fluid how-it-works-screen display-none yes-script" id="how-it-works-screen">
+      <button type="button" class="close" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+      <div class="row row-1">
+        <div class="col-sm-3 how-it-works-col-1">
+          <img class="how-it-works-image" src="images/how_it_works-1.svg">
+        </div>
+        <div class="col-sm-3 how-it-works-col-2">
+          <img class="how-it-works-image" src="images/how_it_works-2.svg">
+        </div>
+        <div class="col-sm-3 how-it-works-col-3">
+          <img class="how-it-works-image" src="images/how_it_works-3.svg">
+        </div>
+        <div class="col-sm-3 how-it-works-col-4">
+          <img class="how-it-works-image" src="images/how_it_works-4.svg">
+        </div>
+      </div>
+      <div class="row row-2">
+        <div class="col-sm-3 how-it-works-col-1">
+          <p>
+            {{i18n.index.how-it-works-1}}
+          </p>
+        </div>
+        <div class="col-sm-3 how-it-works-col-2">
+          <p>
+            {{i18n.index.how-it-works-2}}
+          </p>
+        </div>
+        <div class="col-sm-3 how-it-works-col-3">
+          <p>
+            {{i18n.index.how-it-works-3}}
+          </p>
+        </div>
+        <div class="col-sm-3 how-it-works-col-4">
+          <p>
+            {{i18n.index.how-it-works-4}}
+          </p>
+        </div>
+      </div>
+    </div>
+
+    <div class="screen landing-screen" id="landing-screen">
+      <div class="clearfix"></div>
+
+      <div class="wrapper top-wrapper">
+        <div class="heading">
+          <h1>{{i18n.index.title}}</h1>
+          <h2>{{i18n.index.description}}</h2>
+        </div>
+
+        <div class="how-it-works yes-script">
+          <button class="green-btn track-click" data-track-category="Navigation" data-track-event="How it works" id="how-it-works-button">{{i18n.index.how-it-works}}</button>
+        </div>
+      </div>
+
+      <form id="file-form" class="yes-script" onsubmit="return false;">
+        <input id="file-form-input" type="text" inputmode="url" name="file" placeholder="{{i18n.index.input-placeholder}}" autofocus><button type="submit" class="green-btn">{{i18n.index.go-button}}</button>
+        <div class="container-fluid display-none" id="file-form-alert">
+          <span class="glyphicon glyphicon-warning-sign"></span><span id="file-form-alert-placeholder">Alert</span>
+        </div>
+      </form>
+
+      <noscript>
+        <div class="no-script" >
+          <h2>{{i18n.index.no-script}}</h2>
+        </div>
+      </noscript>
+
+      <div class="attribution" id="attribution">
+        <noscript>
+          ""<a href="https://commons.wikimedia.org/wiki/User:The_Photographer">The Photographer</a>, <a href="https://commons.wikimedia.org/wiki/File:A_day_of_fishing_aground.jpg">„A day of fishing aground“</a>, <a href="https://creativecommons.org/publicdomain/zero/1.0/legalcode">CC0 1.0</a>"
```

Comment:
there should probably be a single opening quote at the beginning of the line here